cal-itp/benefits
# Refactor: Use WebP instead of PNGs

closes part of #183 

This PR is the last of 3 PRs that address performance fixes and refactors for Benefits.

## To Do
- Test in iOS Safari
- Test in Safari
- Test in all other browsers (Android)
- Update image file

## What this PR does

### Styles and images
- Replace a PNG in the background with a WebP one set as an `<img>`, while preserving the CSS
- Adds a CSS class with the same CSS, applied to the `<img>`, to replace the in-line styles
- Change the PNG on the Success! page to the same exact WebP image
- Adds corresponding `heights` to this -- having both a width and height lessens the chance of elements shifting around when the page loads
![image](https://user-images.githubusercontent.com/3673236/144505807-b6ca61fc-c3e0-4744-beb1-0567da644965.png)


## How to test this PR

1. The main image in Desktop and Mobile - Ensure that this image scales up and down correctly
2. Ensure there is an alt tag on the image
